types/mongoose/index.d.ts

```diff
@@ -112,11 +75,9 @@ declare module "mongoose" {
    * Options passed take precedence over options included in connection strings.
    * @returns pseudo-promise wrapper around this
```

Comment:
This returns a `Promise` now (or `null`).

Reply:
You might've commented on an older commit - the latest one already has this defined as follows:

```typescript
  /**
   * Opens the default mongoose connection.
   * Options passed take precedence over options included in connection strings.
   * @returns pseudo-promise wrapper around this
   */
  export function connect(uris: string, options: ConnectionOptions, callback: (err: mongodb.MongoError) => void): null;
  export function connect(uris: string, callback: (err: mongodb.MongoError) => void): null;
  export function connect(uris: string, options?: ConnectionOptions): Promise<Mongoose>;
```

Reply:
Oh my bad, this is fine. I got confused because I thought "pseudo-promise wrapper" referred to `MongooseThenable`, which was deprecated.